src/reportSources/legacyFflogs/eventAdapter/assignOverheal.ts

```diff
@@ -0,0 +1,70 @@
+import {Event, Events} from 'event'
+import {FflogsEvent, HealEvent} from 'fflogs'
+import {AdapterStep} from './base'
+
+/**
+ * FFLogs models damage or healing events as "calculateddamage" or "calculatedheal" events at the time of cast (e.g. Character prepares Action on Target Amount)
```

Comment:
Should probably either amend the comment to just talk about healing, or add logic to (and rename) the adapter to handle both.

Reply:
Whoops. Missed that when I changed course to not worry about overkill after all since nothing wants it at this point. And the logic/signature looked a lot uglier trying to deal with both.